lib/main.dart

```diff
@@ -13,12 +18,38 @@ class MyApp extends StatelessWidget {
   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      title: 'helpwave',
-      theme: lightTheme,
-      darkTheme: darkTheme,
-      themeMode: ThemeMode.dark,// TODO make switchable or use ThemeMode.system
-      home: const LandingPage(),
+    return MultiProvider(
+      providers: [
+        ChangeNotifierProvider(
+          create: (_) => ThemeModel(),
+        ),
+        ChangeNotifierProvider(
+          create: (_) => LanguageModel(),
+        ),
+      ],
+      child: Consumer2<ThemeModel, LanguageModel>(
+        builder:
+            (_, ThemeModel themeNotifier, LanguageModel languageNotifier, __) {
+          return MaterialApp(
+            title: 'helpwave',
+            theme: lightTheme,
+            darkTheme: darkTheme,
+            themeMode: themeNotifier.isDark ? ThemeMode.dark : ThemeMode.light,
+            localizationsDelegates: const [
+              AppLocalizations.delegate,
+              GlobalMaterialLocalizations.delegate,
+              GlobalWidgetsLocalizations.delegate,
+              GlobalCupertinoLocalizations.delegate,
+            ],
+            supportedLocales: const [
+              Locale('en', 'US'),
+              Locale('de', 'DE'),
+            ],
```

Comment:
create a (const) tupel of languages with attributes which than is mapped to the locals or the strings needed for the setting_list_view e.g ("en","US","english") so changes only need to be made in one file